sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -81,9 +81,11 @@
   "sideEffects": false,
   "autoPublish": false,
   "dependencies": {
+    "@azure/abort-controller": "^1.0.0",
     "@azure/core-asynciterator-polyfill": "^1.0.0",
     "@azure/core-auth": "^1.1.3",
-    "@azure/core-http": "^1.2.0",
+    "@azure/core-client": "1.0.0-beta.1",
+    "@azure/core-https": "1.0.0-beta.1",
```

Comment:
this would need to be beta.2 if you want to use `delay`, right?

Reply:
Updated to beta.2